python_ta/reporters/node_printers.py

```diff
@@ -614,6 +614,31 @@ def render_pep8_errors_e306(msg, _node, source_lines=None):
     yield from render_context(msg.line, msg.line + 2, source_lines)
 
 
+def render_missing_return_statement(msg, _node, source_lines=None):
```

Comment:
Note: we use `node` instead of `_node` if the parameter is actually being used in the function body.